admin_manual/enterprise_external_storage/windows-network-drive_configuration.rst

```diff
@@ -137,3 +137,58 @@ In openSUSE, modify the ``/usr/sbin/start_apache2`` file::
 
 Restart Apache, open your ownCloud Admin page and start creating SMB/CIFS 
 mounts.
+
+
+=================
+SMB Notifications
+=================
+
+The SMB protocol allows to register for notifications on filechanges on
+the remote storage server. Notifications are a performant strategy to
+detect changes made directly on the files.
+
+The ownCloud server needs to know about changes of files on integrated storages, so that the changed files will be synced to desktop sync clients. While files changed through ownCloud web or sync clients also get updated their metadata in the filecache, this is not possible if files get changed directly on the storage.
+Whitout notifications polling for changes is needed, but scanning the whole SMB storage is not performant.
+
+On the server commandline a notification listener can be started in an
+extra thread. The listener marks changed files then in the filecache and
+a backgroundjob updates the metadata.
+
+Note: Make sure your background jobs are running as cronjobs.
```

Comment:
I don't think this is needed. The marked folder should be scanned on access anyway.